common/web/gesture-recognizer/src/test/auto/headless/gestures/gestureMatcher.spec.ts

```diff
@@ -69,149 +46,6 @@ function dummyInheritanceMatcher(inheritanceType: PathInheritanceType): gestures
   }
 }
 
-function mockedPredecessor<Type>(
-  lastSample: InputSample<Type>,
-  baseItem?: Type
-): MockedPredecessor<Type> {
-  return {
-    comparisonStandard: {
-      sample: lastSample,
-      baseItem: baseItem ?? lastSample.item
-    },
-    pathMatchers: [],
-    _result: {
-      action: {
-        item: baseItem ?? lastSample.item
-      }
-    }
-  };
-}
-
-function simulateComplexGestureSource<Type>(
```

Comment:
The name of this helper function obviously needed changing.

At the same time... I'll be wanting to reuse this helper method for other unit tests, so I decided to split it, and its helpers, off into their own resource-script.  That's the reason behind the bulk-deletion in this file.